CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.5.3
+* Update the [Custom Brimcap Configuration](https://github.com/brimdata/brimcap/wiki/Custom-Brimcap-Config) wiki article to reflect Zui's new "plugin" directory for pcap index (#319)
```

Comment:
This is a change to unversioned documentation so I don't think it belongs in here.
```suggestion
```

Reply:
I don't see a down side of informing our users of changes in documentation, but I'll accept the suggestion because I think shipping software is more important to me than debating this topic.